lib/remove-console-statements.js

```diff
@@ -0,0 +1,39 @@
+const cleanConsoleLogs = (j, root) => {
+  root.find(j.CallExpression, {
```

Comment:
Seems like this could all live in a function findConsoleCalls or findVariableCalls('console').

Independently I'm a bit concerned that

let console = whatever();
console.foo();

is going to get replaced. I'd like us to have a failing test with that in it, so we can fix it in the future.

What I found when using the previous eslint was that the only allowed external reference in an mjs file was to window (I'm sure there were others but I don't know what they were). I couldn't call requestAnimationFrame, I had to call window.requestAnimationFrame. So we can enforce that with lint and then say that actually this removes calls to window.console.foo and if you break that lint rule, you're on your own.

Reply:
This is something we may need to figure out going forward, but there is something to be said about this script only being run on code we write, so we could just enforce a standard in our own development that you shouldn't do things like reassign `console`

Reply:
Yeah, I think supporting this kind of uncooperative code is a "v2" feature. If anything we could require more explicit author intent, e.g. comments to opt-out of transformation in such cases.

Reply:
I think there must be an easy-ish way to do this because eslint can tell the difference between

```js
console.log("foo");
```

and

```js
const console = null;
console.log("foo");
```